core/src/main/java/org/jdbi/v3/core/mapper/EnumMapper.java

```diff
@@ -42,50 +39,7 @@
      * @return an enum mapper that matches on {@link Enum#ordinal()}
      */
     public static <E extends Enum<E>> ColumnMapper<E> byOrdinal(Class<E> type) {
-        return new ByOrdinal<>(type);
+        return new EnumByOrdinalMapper<>(type);
     }
 
-    private static class ByName<E extends Enum<E>> extends EnumMapper<E> {
```

Comment:
This was all nicely localized in one easily read file.  Now it's in five :(

Reply:
I'm not a fan of lobbing things together for non-technical reasons? Single responsibility, minimalism, yadda yadda?

Reply:
I don't see this approach as particularly minimal, unfortunately.  I'm taking a pass at reducing some of the copy+paste.